src/ui/api/project.go

```diff
@@ -118,6 +118,29 @@ func (p *ProjectAPI) Post() {
 		}
 		return
 	}
+
+	user, err := dao.GetUser(models.User{
+		UserID: p.userID,
+	})
+	if err != nil {
+		log.Errorf("failed to get user by ID %d: %v", p.userID, err)
+		p.CustomAbort(http.StatusInternalServerError, http.StatusText(http.StatusInternalServerError))
+	}
+
+	accessLog := models.AccessLog{
```

Comment:
Do we need an rest api for access log? Then it can call the post to create log here.